src/ops/fuelup_default.rs

```diff
@@ -46,7 +46,7 @@ pub fn default(toolchain: Option<String>) -> Result<()> {
         for toolchain in toolchains {
             info!("{}", toolchain);
         }
-        // TODO: we should consider migrating to use thiserror in order to return a custom error here
```

Comment:
thiserror is a rust crate and is written like this on purpose. Can we revert this. Happy to approve after this change is removed.

Reply:
done